tensornetwork/backends/jax/jax_backend_test.py

```diff
@@ -530,3 +531,18 @@ def test_matrix_ops_raises(dtype, method):
   matrix = backend.randn((4, 3), dtype=dtype, seed=10)
   with pytest.raises(ValueError, match=r".*N\*N matrix.*"):
     getattr(backend, method)(matrix)
+
+
+def test_jit():
+  backend = jax_backend.JaxBackend()
+
+  def fun(x, A, y):
+    return jax.numpy.dot(x, jax.numpy.dot(A, y))
+
+  fun_jit = backend.jit(fun)
+  x = jax.numpy.array(np.random.rand(4))
+  y = jax.numpy.array(np.random.rand(4))
+  A = jax.numpy.array(np.random.rand(4, 4))
+  res1 = fun(x, A, y)
+  res2 = fun_jit(x, A, y)
```

Comment:
Be sure to also add a test that uses key word args.

Reply:
was just about to push :)